content-src/asrouter/templates/SimpleSnippet/_SimpleSnippet.scss

```diff
@@ -88,4 +91,44 @@
   .icon {
     align-self: flex-start;
   }
+
+  &.has-section-header .innerWrapper {
+    // account for section header being 100% width
+    flex-wrap: wrap;
```

Comment:
Can you also set `padding-top: 7px;`? I think that matches up with the visual comp (the annotation is off)